src/equations/1d/compressible_euler.jl

```diff
@@ -0,0 +1,838 @@
+  end
+
+  return SVector(f1, f2, f3)
+end
+
+
+# Determine maximum stable time step based on polynomial degree and CFL number
+function calc_max_dt(u, element_id, invjacobian, cfl,
+                     equation::CompressibleEulerEquations1D, dg)
+  λ_max = 0.0
+  for i in 1:nnodes(dg)
+    rho, rho_v1, rho_e = get_node_vars(u, dg, i, element_id)
+    v1 = rho_v1 / rho
+    v_mag = abs(v1)
+    p = (equation.gamma - 1) * (rho_e - 1/2 * rho * v_mag^2)
+    c = sqrt(equation.gamma * p / rho)
+    λ_max = max(λ_max, v_mag + c)
+  end
+
+  dt = cfl * 2 / (nnodes(dg) * invjacobian * λ_max)
+
+  return dt
+end
+
+
+# Convert conservative variables to primitive
+@inline function cons2prim(u, equation::CompressibleEulerEquations1D)
+  rho, rho_v1, rho_e = u
+
+  v1 = rho_v1 / rho
+  p = (equation.gamma - 1) * (rho_e - 0.5 * rho * (v1^2))
+
+  return SVector(rho, v1, p)
+end
+
+
+# Convert conservative variables to entropy
+@inline function cons2entropy(u, equation::CompressibleEulerEquations1D)
+  rho, rho_v1, rho_e = u
+
+  v1 = rho_v1 / rho
+  v_square = v1^2
+  p = (equation.gamma - 1) * (rho_e - 0.5 * rho * v_square)
+  s = log(p) - equation.gamma*log(rho)
+  rho_p = rho / p
+
+  w1 = (equation.gamma - s) / (equation.gamma-1) - 0.5 * rho_p * v_square
+  w2 = rho_p * v1
+  w3 = -rho_p
+
+  return SVector(w1, w2, w3)
+end
+
+
+# Convert primitive to conservative variables
+@inline function prim2cons(prim, equation::CompressibleEulerEquations1D)
+  rho, v1, p = prim
+  rho_v1 = rho * v1
+  rho_e  = p/(equation.gamma-1) + 0.5 * (rho_v1 * v1)
+  return SVector(rho, rho_v1, rho_e)
+end
+
+
+#@inline function density(u, equation::CompressibleEulerEquations2D)
+#  rho = u[1]
+#  return rho
+#end
+
+
+#@inline function pressure(u, equation::CompressibleEulerEquations2D)
+#  rho, rho_v1, rho_v2, rho_e = u
+#  p = (equation.gamma - 1) * (rho_e - 0.5 * (rho_v1^2 + rho_v2^2) / rho)
+#  return p
+#end
+
+
+#@inline function density_pressure(u, equation::CompressibleEulerEquations2D)
+#  rho, rho_v1, rho_v2, rho_e = u
+#  rho_times_p = (equation.gamma - 1) * (rho * rho_e - 0.5 * (rho_v1^2 + rho_v2^2))
+#  return rho_times_p
+#end
+
+
+# Convert conservative variables to indicator variable for discontinuities (elementwise version)
+@inline function cons2indicator!(indicator, cons, element_id, n_nodes, indicator_variable,
+                                 equation::CompressibleEulerEquations1D)
+  for i in 1:n_nodes
+    indicator[1, i] = cons2indicator(cons[1, i, element_id], cons[2, i, element_id],
+                                        cons[3, i, element_id],
+                                        indicator_variable, equation)
+  end
+end
+
+
+# Convert conservative variables to indicator variable for discontinuities (pointwise version)
+@inline function cons2indicator(rho, rho_v1, rho_e, ::Val{:density},
+                                equation::CompressibleEulerEquations1D)
```

Comment:
I think these methods are changed in the current master. You should update your master from the main Trixi repo master and then merge master into your branch. Then have a look at how this part of `compressible_euler_equations.jl` looks in 2D now.